scipy/interpolate/interpnd.pyx

```diff
@@ -909,12 +946,28 @@ class CloughTocher2DInterpolator(NDInterpolatorBase):
 
     def __init__(self, points, values, fill_value=np.nan,
                  tol=1e-6, maxiter=400, rescale=False):
+        self.tol = tol
+        self.maxiter = maxiter
```

Comment:
New attributes, should be removed or motivated.

Reply:
These two attributes are used in the `_set_values` function of the `CloughTocher` class, which is an override for the one of `NDInterpolatorBase`, hence they are saved here and used afterwards.

Reply:
We should not add public attributes unless really necessary.
Consider either not attaching them to the instance (like it is on main), or adding a leading underscore to mark them as private, like `self._tol`

Reply:
Fixed.